src/lib/system-info.ts

```diff
@@ -75,7 +75,7 @@ export async function getCpuId(): Promise<string | undefined> {
 	}
 }
 
-const undervoltageRegex = /under.*voltage/;
+const undervoltageRegex = /[U|u]nder.*voltage/;
```

Comment:
what about this or some other flag to make the test case insensitive rather then making the pattern more complex.

```suggestion
const undervoltageRegex = new RegExp('under.*voltage', 'i');
```